ui/pages/swaps/build-quote/build-quote.js

```diff
@@ -367,6 +368,11 @@ export default function BuildQuote({
     }
   }
 
+  const swapYourTokenBalance = t('swapYourTokenBalance', [
```

Comment:
Are these arguments backward?  `$1 is a token symbol and $2 is the max. number of decimals allowed for the token`

Reply:
This is an existing translation, which I just abstracted here since it was used multiple times.

```
"swapYourTokenBalance": {
    "message": "$1 $2 available to swap",
    "description": "Tells the user how much of a token they have in their balance. $1 is a decimal number amount of tokens, and $2 is a token symbol"
  },
```